Intersect.Server/Entities/Player.cs

```diff
@@ -1414,15 +1414,17 @@ public override int CalculateAttackTime()
             //Add up player equipment values
             for (var i = 0; i < Options.EquipmentSlots.Count; i++)
             {
-                if (Equipment[i] >= 0 && Equipment[i] < Options.MaxInvItems)
+                var equipment = Equipment[i];
+                if (equipment >= 0 && equipment < Items.Count && equipment < Items.Count)
```

Comment:
Same check is being done twice or am I misreading this?

Reply:
Yeah..... fixed lol.